src/status_im/ui/screens/communities/community_overview_redesign.cljs

```diff
@@ -3,70 +3,137 @@
-      :style  {:height 180
+      :style  {:position :relative
                :flex 1}}]]
-   [react/view (styles/card-view-content-container 20)
-    [react/view (styles/card-view-chat-icon 80)
-     [communities.icon/community-icon-redesign community 80]]
-    (when (= status :gated)
-      [react/view (styles/permission-tag-styles)
-       [community-view/permission-tag-container
-        {:locked       locked
-         :status       status
-         :tokens       tokens}]])
-    [community-view/community-title
-     {:title name
-      :size :large
-      :description description}]
-    [community-view/community-stats-column :card-view]
-    [community-view/community-tags tags]
-    [preview-list-component]
-    (when (not joined)
-      [quo2.button/button
-       {:on-press  #(>evt [:bottom-sheet/show-sheet
-                           {:content (fn [] [request-to-join/request-to-join community])
-                            :content-height 300}])
-        :style
-        {:width "100%"
-         :margin-top 20
-         :margin-left :auto
-         :margin-right :auto}
-        :before :main-icons/community}
-       (i18n/label :join-open-community)])
+   [react/view {:flex               1
+                :height 20
+                :border-radius      16
+                :background-color (colors/theme-colors
+                                   colors/white
+                                   colors/neutral-90)}
+    [react/view {:padding-horizontal 20}
+     [react/view {:border-radius    40
+                  :border-width 1
+                  :border-color colors/white
+                  :position         :absolute
+                  :top              (- (/ 80 2))
+                  :left             (/ 70 4)
+                  :padding          2
+                  :background-color (colors/theme-colors
+                                     colors/white
+                                     colors/neutral-90)}
+      [communities.icon/community-icon-redesign community 80]]
+     (when (and (not joined)
+                (= status :gated))
+       [react/view (styles/permission-tag-styles)
+        [community-view/permission-tag-container
+         {:locked       locked
+          :status       status
+          :tokens       tokens}]])
+     (when joined
+       [react/view {:position         :absolute
+                    :top              12
+                    :right            12}
+        [quo2.status-tags/status-tag {:status {:type :positive} :label (i18n/label :joined)}]])
+     [community-view/community-title
+      {:title name
+       :size :large
+       :description description}]
+     [community-view/community-stats-column :card-view]
+     [community-view/community-tags tags]
+     [preview-user-list]
+     (when (not joined)
+       [quo2.button/button
+        {:on-press  #(>evt [:bottom-sheet/show-sheet
+                            {:content (fn [] [request-to-join/request-to-join community])
```

Comment:
More like FYI, you can probably replace the anonymous function with `(constantly [request-to-join/request-to-join community])`.